README.md

```diff
@@ -1,7 +1,7 @@
 # flux-get-started
 
 We published a step-by-step run-through on how to use Flux and Helm Operator [over
-here](https://github.com/weaveworks/flux/blob/master/site/helm-get-started.md).
+here](https://github.com/fluxcd/flux/blob/master/docs/install/helm-get-started.md).
```

Comment:
This link is going to change very soon due to https://github.com/fluxcd/flux/pull/2313

Reply:
@hiddeco I'll get this merged and I'll update the links once more after the docs restructure.